src/workerd/api/pyodide/pyodide.c++

```diff
@@ -770,10 +768,18 @@ kj::Promise<void> loadPyodidePackage(const PythonConfig& pyConfig,
   }
 
   // Need to fetch from network
+  kj::HttpHeaderTable table;
+  kj::TlsContext::Options tlsOptions;
+  tlsOptions.useSystemTrustStore = true;
+  kj::Own<kj::TlsContext> tlsContext = kj::heap<kj::TlsContext>(kj::mv(tlsOptions));
+
+  auto tlsNetwork = tlsContext->wrapNetwork(network);
+  auto client = kj::newHttpClient(timer, table, network, *tlsNetwork);
+
   kj::String url =
       kj::str("https://storage.googleapis.com/cloudflare-edgeworker-python-packages/", path);
```

Comment:
Nit: I know this isn't new here but would really prefer this to be moved to a constant... and ideally made configurable. Hard coding URLs into the source really isn't my favorite approach.